### DISCUSSION TITLE
Support Geography types

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#11410] [Feature Requests]
![image](https://user-images.githubusercontent.com/10562635/152313694-f500620e-6eb2-4a1c-8a45-b8d728a94a91.png)

at the moment directus only supports geometry, with geography you get:
![image](https://user-images.githubusercontent.com/10562635/152313987-ba7349a5-721f-4d1c-ac66-97588ab1c199.png)

**Comment by rijkvanzanten** [comment]
This is technically not specific to MS SQL 🙂 I think we should look into supporting geography in all of the vendors

**Reply by oreilles** [reply]
In PostGIS and SQL Server, I believe the `geography` data type only differs from the `geometry` in the way it is stored, and used in distance calculation (and feature comparisons algorithm). All in all, since we only get and push geometries to the database as `WKT`, I believe we can add an entry in the `localTypeMap` to handle geographies as geometries (there should be no difference in the way we process them).

I don't think we want users to be able to make the distinction between the two when creating new columns though - it's added complexity that most users don't need/understand, plus geography datatype does not even exist in MySQL and Oracle.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
